src/background/services/storage/schemaMigrations/migrations/wallet_v5/wallet_v5.test.ts

```diff
@@ -0,0 +1,379 @@
+import { AddressPublicKey } from '@src/background/services/secrets/AddressPublicKey';
+import { CommonError } from '@src/utils/errors';
+
+import walletV5Migration from './wallet_v5';
+import * as Legacy from './legacyModels';
+import { expectToThrowErrorCode } from '@src/tests/test-utils';
+
+jest.mock('@src/background/services/secrets/AddressPublicKey');
+
+describe('src/background/services/storage/schemaMigrations/migrations/wallet_v5', () => {
+  const mockMnemonicSecrets: Legacy.MnemonicSecrets = {
+    id: 'mnemonic',
+    secretType: 'mnemonic',
+    derivationPath: 'bip44',
+    name: 'Seedphrase Wallet',
+    mnemonic: 'test mnemonic',
+    xpub: 'xpub',
+    xpubXP: 'xpubXP',
+  };
+  const mockLedgerSecrets: Legacy.LedgerSecrets = {
+    id: 'ledgy',
+    derivationPath: 'bip44',
+    name: 'Ledger Wallet',
+    xpub: 'xpub',
+    xpubXP: 'xpubXP',
+    secretType: 'ledger',
+    btcWalletPolicyDetails: {
```

Comment:
Should we have test scenarios where the optional properties are missing?

Reply:
Great suggestion, caught a bug :) 79c17a149

Thanks!